### DISCUSSION TITLE
Is there a way to increase or disable the auto-logoff time?

### DISCUSSION THREAD

**Discussion by bananabrann** [danny-avila/LibreChat#2571] [Q&A] [answered]
Super rudimentary question, but I tried looking through the config as well as searching open issues and discussions on GitHub, but didn't see anything about this 😅 My wife and grandma (I know, she's super advanced) get kicked out and lose their current message and become frustrated with the current timeout 

**Is there a way to increase (or outright disable) the time before you're auto logged out?**

**Comment by fuegovic** [comment, marked as answer]
you need to use https to prevent that behavior

then you can adjust the expiry time:
```sh
SESSION_EXPIRY=1000 * 60 * 15
REFRESH_TOKEN_EXPIRY=(1000 * 60 * 60 * 24) * 7
```

**Reply by bananabrann** [reply]
Ah gotcha, I use HTTP. Looks like we're getting an upgrade, thank you!
